experiment/coverage_utils.py

```diff
@@ -292,3 +406,170 @@ def extract_covered_regions_from_summary_json(summary_json_file):
     except Exception:  # pylint: disable=broad-except
         logger.error('Coverage summary json file defective or missing.')
     return covered_regions
+
+
+def prepare_name_dataframes(df_container):
+    """Populating DataFrames with experiment specific
+    benchmark names, file names, and function names."""
+
+    # Collect segment and function CSV for all benchmark-fuzzer combinations.
+    try:
+        # Separate file names into a different CSV file and allocate ID's
+        # for each unique file name.
+        df_container.filename_df['file_name'] = \
```

Comment:
We generally don't use `\` (frankly I consider almost any use of it bad).
Can you please use paren to split up lines instead?